### ISSUE TITLE
Rule Proposal: the style attribute is not allowed in templates

### ISSUE THREAD

**Issue by l-zoy** [vuejs/eslint-plugin-vue#1477]
**Please describe what the rule should do:**

templates prohibit the use of style attributes unless they require dynamic binding

**What category should the rule belong to?**

[x] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<template>
  <div style="color: red"></div> <!-- bad -->
  <div :style="'color:' + color"></div> 
</template>
<script>
export default {
  data() {
    return {
      color: 'red'
    }
  }
}
</script>

```

**Additional context**

If I can, I am happy to implement it 😆

**Comment by ota-meshi** [comment]
Perhaps you can use the following rule:
https://eslint.vuejs.org/rules/no-static-inline-styles.html

**Comment by ota-meshi** [comment]
You can use the `vue/no-static-inline-styles` rule.
